### ISSUE TITLE
[HIPRI] China, SiAR Radar, further update

### ISSUE THREAD

**Issue by Luigi-II** [PygmalionOfCyprus/cmo-db-requests#5401]
### DB Selector

DB3K

### Affected DBID(s)

Sensor 7418

### Summary of Changes

- [ ] Change the **RadarProcessingGainLoss** to **5**. Same as SLC-18.
- [ ] Change the **RadarSystemNoiseLevel** to **1**. Slightly better than SLC-18. Consistent with the statement of "improving the signal-to-noise ratio".
- [ ] Change the **RadarVerticalBeamwidth** to **1**. Same as SLC-18.
- [ ] Add **Active Electronically Scanned Array (AESA)** and **Frequency Agile** to its properties. MIMO radar is a more advanced AESA system.

At present, SIAR radar's performance in the DB is just (or even worse than) ordinary OTH-B radar. In terms of both anti-stealth and regular target detection capability, it is even far worse than most of the ordinary long-range surveillance radars (for example, the detection capability of stealth targets is only half of the level of AN/TPS-77 of the 1980s tech). 

As a "neo-system" AESA anti-stealth radar, it is reasonable that its sensor's other indicators should be at least no worse than SLC-18, which operates in similar frequency and are also advanced AESA (though the system is not as advanced as MIMO).

To put it another way, China has been rolling out various anti-stealth radars at the Zhuhai Air Show, but there has never been a similar product like SIAR-MIMO radar. That seems to explain the radar's technical advancement and secrecy.

### Sources

https://en.wikipedia.org/wiki/MIMO_radar

> MIMO radar is **an advanced type of phased array radar** employing digital receivers and waveform generators distributed across the aperture. 

https://www.radartutorial.eu/02.basics/MIMO-radar.en.html

> By **improving the signal-to-noise ratio**, the probability of detection of the targets is also increased.

**Comment by PygmalionOfCyprus** [comment]
Handled 515.
